src/app/services/toast/toast.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from '@angular/core';
+import { ToastController } from '@ionic/angular';
+
+export type PredefinedColors =
+  | 'primary'
+  | 'secondary'
+  | 'tertiary'
+  | 'success'
+  | 'warning'
+  | 'danger'
+  | 'light'
+  | 'medium'
+  | 'dark';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ToastService {
+  constructor(private toastController: ToastController) {}
+
+  async presentToast(
+    message: string,
+    duration: number = 2000,
+    position: 'top' | 'middle' | 'bottom' = 'bottom',
+    color: PredefinedColors = 'success'
```

Comment:
Pode inverter a ordem dos parâmetros menos utilizados para o fim?
assim não precisamos passar undefined na duração, já que é algo que não vai ser atualizado com tanta frequência
```suggestion
    message: string,
    position: 'top' | 'middle' | 'bottom' = 'bottom',
    duration: number = 2000,
    color: PredefinedColors = 'success'
```